agent/bench-scripts/postprocess/process-iteration-samples

```diff
@@ -261,6 +282,8 @@ foreach my $metric_name (sort keys %iteration) {
 		for ($i = 0; $i < scalar @{$iteration{$metric_name}{$metric_type}}; $i++) {
 			my $key;
 			foreach $key (sort keys %{$iteration{$metric_name}{$metric_type}[$i]}) {
+				#my $sample_value = $iteration{$metric_name}{$metric_type}[$i]{$key};
+				#print "[$metric_name][$metric_type][$i][$key] = $sample_value \n";
```

Comment:
Here too: do we want to keep the debugging output?

Reply:
Removed.